docs/admin/system-information.rst

```diff
@@ -1729,6 +1727,12 @@ After you reindexed all tables, this cluster check will pass.
 License check
 .............
 
+
+.. NOTE::
```

Comment:
@mechanomi I removed the check, but kept the section because of the links in the admin-ui to the sections.

Any suggestions how to phrase the note?

Reply:
left a note. sorry I messed up and my reply is in a separate comment